src/pages/products/products.jsx

```diff
@@ -30,48 +49,142 @@ return(
                     </div>
                     <h3 className="margin-top">Price</h3>
                     <div className="space-between">
-                        <span>50</span>
-                        <span>100</span>
-                        <span>150</span>
+                        <span>200</span>
+                        <span>1000</span>
+                        <span>2000</span>
                     </div>
 
-                    <input type="range" className="range"/>
-                    <label htmlFor=""></label>
+                    
+                    <label htmlFor="priceRange">                       
+                        <input 
+                        type="range" 
+                        className="range"
+                        id="priceRange"
```

Comment:
Make another folder as Filters and move the Filters part over there, just a suggestion

Reply:
sure.